### ISSUE TITLE
Add welcome/intro screens

### ISSUE THREAD

**Issue by anandwana001** [fossasia/loklak_wok_android#84]
Showing welcoming screen when user first time opens the app

1. Posting tweet from the app
2. Default tweet cards

**Comment by vibhcool** [comment]
it sounds awesome 👍 , what do you mean by default tweet cards mean, could you eplain it a bit :) ?

**Comment by anandwana001** [comment]
@vibhcool the harvested tweets we are getting when first open the app.

**Comment by anandwana001** [comment]
@mariobehling if you could give some feedback on this issue! 
As in SUSI Android we have intro slides, we are thinking to have for loklak_wok also where we can show the features.

**Comment by bipinkumar919** [comment]
Any progress on this. May I do it if yet not implemented?

**Comment by anandwana001** [comment]
@bipinkumar919 I am working on it, because of exams it skipped it but back on it from now. I am just waiting for someone like @mariobehling or @vibhcool to give their suggestions on it because we also need to make some images to show there.

**Comment by bipinkumar919** [comment]
OK cool.

**Comment by vibhcool** [comment]
Yes, you can proceed :+1: , I missed this comment 😅 you may ping on gitter if someone misses your comment
